spec.html

```diff
@@ -119,7 +119,7 @@ <h1>AggregateError ( _errors_, _message_ )</h1>
         <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
+          1. Let _O_ be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateErrorPrototype%"`, « [[ErrorData]], [[AggregateErrors]] »).
```

Comment:
This is intentional and not a mistake; the "dot" form of intrinsics is preferred for anything reachable off of an intrinsic.

Reply:
@marjakh For reference, see https://tc39.es/ecma262/#sec-well-known-intrinsic-objects.

> Within this specification a reference such as %name% means the intrinsic object, associated with the current realm, corresponding to the name. A reference such as %name.a.b% means, as if the "b" property of the "a" property of the intrinsic object %name% was accessed prior to any ECMAScript code being evaluated.

Reply:
Ah, cool, thanks for the info :)